### DISCUSSION TITLE
Notes preview

### DISCUSSION THREAD

**Discussion by schaitraQuicken** [deejay-hub/timeline-lwc#187] [Q&A] [answered]
clicking on a Note object in the timeline, opens the preview as an overly but is just a black screen. Any setting to fix this
![2022-05-17_12-44-24](https://user-images.githubusercontent.com/49919378/168865451-ef63f645-921d-4769-9d12-00d217f4b36a.jpg)

**Comment by deejay-hub** [comment, marked as answer]
Yes that's correct. You will need to click the 'Open Note' button in the middle of the overlay to navigate to the Note record. Lightning Web Component support for opening Notes any other way isn't available currently.
